backend/app/controllers/validations/reservations/create_reservation.rb

```diff
@@ -0,0 +1,14 @@
+class Validations::Reservations::CreateReservation < Validations::BaseValidator
+
+  ATTR_LIST = %i[phone_number table_size date ]
+
+  attr_accessor(*ATTR_LIST)
+
+  validates :phone_number, :table_size, :date, presence: true
```

Comment:
Write methods to validate these instead:
1. phone_number -> to be a string and match with a regex
2. table_size -> to be an integer greater than 0
3. date -> to be a date (check for some conversion techniques instead of regex)